lib/events.js

```diff
@@ -258,6 +258,9 @@ exports.setApprovalStatus = async (req, res) => {
     await sequelize.transaction(async (t) => {
         await req.event.update({ status: req.body.status }, { transaction: t });
 
+        req.event.organizers = await Promise.all(req.event.organizers.map((organizer) =>
```

Comment:
this one will also make a lot of requests if the organizers's amount is too big, same way as with the massmailer

Reply:
Yeah, but we already do this when we update events though. And there are normally not a lot of organizers. I will change this with the discussed new endpoint later

Reply:
makes sense